cmd/internal/cli/pull.go

```diff
@@ -79,7 +79,7 @@ var pullNameFlag = cmdline.Flag{
 	Name:         "name",
 	Hidden:       true,
 	Usage:        "specify a custom image name",
-	EnvKeys:      []string{"NAME"},
+	EnvKeys:      []string{"PULL_NAME"},
```

Comment:
@cclerget, maybe `SINGULARITY_NAME` was intentional? Was it supported to interfere with the other `SINGULARITY_NAME`?

Reply:
These variables should not collide

Reply:
:+1: